OG/OG/gen/OGListener.cs

```diff
@@ -8,7 +8,7 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
-// Generated from /Users/saxjax/developer/P4-project/OG/OG/OG.g4 by ANTLR 4.9.1
+// Generated from C:/Users/Martin/projects/P4/P4-project/OG/OG\OG.g4 by ANTLR 4.9.1
 
 // Unreachable code detected
 #pragma warning disable 0162
```

Comment:
hvad er det for et state som er blevet disablet?

Reply:
Godt spørgsmål? De der pragma warnings kommer fra ANTLRs auto generering, så ved det ikke :-P